src/components/ComponentPages/ProfileInfo/ProfileInfoUI/index.tsx

```diff
@@ -0,0 +1,268 @@
+import React from "react";
+import { Row, Col, Typography, Form, Input, Button, Select, DatePicker, Radio } from "antd";
+import styles from "./ProfileInfo.module.scss";
+import messages from "../../../../messages";
+
+const { Title, Text } = Typography;
+const { Option } = Select;
+
+function ProfileInfoUI() {
+
+  const config = {
+    rules: [
+      {
+        type: 'object',
+        required: true,
+        message: 'Please select time!',
+      },
+    ],
+  };
+  const selectAfter = (
+    <Select defaultValue="private" className="select-after">
+      <Option value="private">Private</Option>
+      <Option value="public">Public</Option>
+    </Select>
+  );
+
+
+  return (
+    <section className={styles.profileInfo_wrapper}>
+      <Form
+        name="verifyNumber"
+        initialValues={{ prefix: "+91" }}
+        layout="vertical"
+      >
+        <div className={styles.section_one}>
+          <Row gutter={30}>
+            <Col md={12}>
+              <Form.Item
+                name="phoneNumber"
+                label={messages.labels.phoneNumber}
+                rules={[
+                  {
+                    required: true,
+                    message: 'Please Enter Phone Number!',
+                  },
+                ]}
+              >
+
+                <Input placeholder={messages.placeholders.phoneNumber} size="large" />
+              </Form.Item>
+            </Col>
+            <Col md={12}>
+              <Form.Item
+                name="mobileCarrier"
+                label={messages.labels.mobileCarrier}
+                rules={[
+                  {
+                    required: true,
+                    message: 'Please Enter Mobile Carrier!',
+                  },
+                ]}
+              >
+
+                <Input placeholder={messages.placeholders.mobileCarrier} size="large"/>
+              </Form.Item>
+            </Col>
+          </Row>
+            <Button
+              type="primary"
+              htmlType="submit"
+              className="ant-btn ant-btn-orange ant-btn-lg"
+            >
+              Verify
+            </Button>
+        </div>
+      </Form>
+
+
+      <Form
```

Comment:
It must come from a component

Reply:
Done

Reply:
You missed the put the profileInfo form into a different component